Resources/public/js/views/ez-navigationhubview.js

```diff
@@ -159,15 +186,37 @@ YUI.add('ez-navigationhubview', function (Y) {
 
             container.setHTML(this.template({
                 user: this.get('user').toJSON(),
+                zones: this.get('zones'),
             }));
+            this._renderNavigationItems();
             this._setNavigationMenu(this.get('activeNavigation'));
             this._uiSetActiveNavigation();
             return this;
         },
 
+        /**
+         * Renders the navigation items in the corresponding navigation zone
+         *
+         * @method _renderNavigationItems
+         * @protected
+         */
+        _renderNavigationItems: function () {
+            var that = this;
+
+            Y.Object.each(this.get('zones'), function (zone, key) {
+                var after = that._getNavigationNode(key).one('.ez-logo');
+
+                Y.Array.each(that.get(key + 'NavigationItems'), function (view) {
```

Comment:
that would not change anything in the code structure. getAttrs would return something like:
```js
{platformNavigationItems: [/* array of items */, studioNavigationItems: [/* array of items */, /* ... */}
```